Plugins/Creature/Creature.cpp

```diff
@@ -192,43 +192,43 @@ ArgumentStack Creature::RemoveFeat(ArgumentStack&& args)
 ArgumentStack Creature::GetKnowsFeat(ArgumentStack&& args)
 {
     ArgumentStack stack;
-    int32_t retval = 0;
+    int32_t retVal = 0;
     if (auto *pCreature = creature(args))
     {
-        const auto feat = Services::Events::ExtractArgument<int32_t>(args);  ASSERT(feat <= 65535);
+        const auto feat = Services::Events::ExtractArgument<int32_t>(args);  ASSERT_OR_THROW(feat <= 65535);
 
-        retval = pCreature->m_pStats->HasFeat(static_cast<uint16_t>(feat));
+        retVal = pCreature->m_pStats->HasFeat(static_cast<uint16_t>(feat));
     }
-    Services::Events::InsertArgument(stack, retval);
+    Services::Events::InsertArgument(stack, retVal);
     return stack;
 }
 
 ArgumentStack Creature::GetFeatCountByLevel(ArgumentStack&& args)
 {
     ArgumentStack stack;
-    int32_t retval = -1;
+    int32_t retVal = -1;
     if (auto *pCreature = creature(args))
     {
-        const auto level = Services::Events::ExtractArgument<int32_t>(args);  ASSERT(level <= 40);
+        const auto level = Services::Events::ExtractArgument<int32_t>(args);  ASSERT_OR_THROW(level <= 40);
 
         if (level > 0 && level <= pCreature->m_pStats->m_lstLevelStats.num)
```

Comment:
huh, I see we already check for `level > 0`, so maybe not necessary. But I think it's still better to give a meaningful error than silently ignore. If you add `>= 1`, you can also remove this.